src/Components/RightSidebar.elm

```diff
@@ -17,7 +18,17 @@ view { showRightSidebarMenu, showRightSidebarMenuMsg, clearPlaylistMsg, refreshP
         column
             [ height (px panelHeight), width (px 400), Background.color Colors.playlistHeaderBackground, alignRight, htmlAttribute <| Html.Attributes.style "pointer-events" "all" ]
             [ row [ width fill ]
-                [ Input.button [ Background.color Colors.backgroundKodi, height (px 50), width (px 100), padding 8 ] { onPress = Just rightSidebarMsg, label = Element.text "Kodi" }
+                [ Input.button [ Background.color Colors.backgroundKodi, height (px 50), width (px 100), padding 8 ]
+                    { onPress = Just rightSidebarMsg
+                    , label =
+                        Element.row []
+                            [ image [ width (px 15), height (px 15), paddingXY 15 0 ]
+                                { description = ""
+                                , src = "http://localhost:1234/logo.png"
```

Comment:
This url needs to be relative, when we build this for kodi (or even in dev) it's not clear, that the image will always be at that port (and host)